src/go/rpk/pkg/cli/profile/create.go

```diff
@@ -131,6 +141,7 @@ func createProfile(
 	fromRedpanda string,
```

Comment:
fwiw I wrote this function terribly to begin with but, it does the job. Originally I had more unit testing ambitions.